tests/layer_tests/tensorflow_tests/test_tf_Inv.py

```diff
@@ -41,4 +41,49 @@ def test_inv_basic(self, params, ie_device, precision, ir_version, temp_dir,
                                          use_legacy_frontend):
         self._test(*self.create_inv_net(**params),
                    ie_device, precision, ir_version, temp_dir=temp_dir,
-                   use_legacy_frontend=use_legacy_frontend)
\ No newline at end of file
+                   use_legacy_frontend=use_legacy_frontend)
+
+class TestComplexInv(CommonTFLayerTest):
+    def _prepare_input(self, inputs_info):
+        rng = np.random.default_rng()
+        assert 'param_real' in inputs_info
+        assert 'param_imag' in inputs_info
+        param_real_shape_1 = inputs_info['param_real']
+        param_imag_shape_1 = inputs_info['param_imag']
+        inputs_data = {}
+        inputs_data['param_real'] = 4 * rng.random(param_real_shape_1).astype(np.float32) - 2
+        inputs_data['param_imag'] = 4 * rng.random(param_imag_shape_1).astype(np.float32) - 2
+        return inputs_data
+
+    def create_complex_inv_net(self, input_shape):
+        import tensorflow as tf
+        tf.compat.v1.reset_default_graph()
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            param_real = tf.compat.v1.placeholder(np.float32, input_shape, 'param_real')
+            param_imag = tf.compat.v1.placeholder(np.float32, input_shape, 'param_imag')
+            complex = tf.raw_ops.Complex(real=param_real, imag=param_imag)
+            inv = tf.raw_ops.Inv(x=complex, name="complex_inv")
+            real = tf.raw_ops.Real(input=inv)
+            img = tf.raw_ops.Imag(input=inv)
+            tf.compat.v1.global_variables_initializer()
+            tf_net = sess.graph_def
+
+        return tf_net, None
+
+    test_data_basic = [
+        dict(input_shape=[]),
+        dict(input_shape=[2]),
+        dict(input_shape=[1, 3]),
+        dict(input_shape=[2, 3, 4]),
+        dict(input_shape=[3, 4, 5, 6]),
+    ]
+    @pytest.mark.parametrize("params", test_data_basic)
```

Comment:
please use parameterize just for `input_shape` instead of `params`:

```
    @pytest.mark.parametrize('input_shape', [[], [2], [1, 3], [2, 3, 4], [3, 4, 5, 6]])
```